app/src/analytics/analytics-client.test.ts

```diff
@@ -0,0 +1,39 @@
+import { isProductionEnv, createAnonymousId, getAccountDetails, sendAnalytics } from './analytics-client';
+
+jest.mock('./analytics-client', () => ({
+    ...jest.requireActual('./analytics-client'),
+    getAnalyticsClient: jest.fn().mockReturnValue({ cats: 'ssss' })
+}));
+
+jest.mock('@atlassiansox/analytics-node-client');
+describe('isProductionEnv', () => {
```

Comment:
Oh my god my eyes 😱 Where are the new lines 😭 (/nit :lol:)

Reply:
lolol space added